pootle/static/css/style.css

```diff
@@ -2121,15 +2127,19 @@ span.email .i
  * Import-export styling
  */
 
-#import-export-sidebar {
-    background-color: #e5e5e5;
-    padding: 2em 1em;
+#import-export-sidebar
+{
+    margin: 0;
+    padding: 0;
 }
 
-#upload-form {
-    margin-top: .5em;
+.action-link
+{
+    display: block;
```

Comment:
why not make it more intuitive and wrap the inline element within a block element such as `<ul><li>`, `<p>`, or `<div>`?